lib/update-globals.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const fs = require('fs');
+const vm = require('vm');
+const path = require('path');
+const https = require('https');
+
+// collect globals
+const jsGlobals = {};
+
+vm.createContext(jsGlobals);
+vm.runInContext(`
+  this.props = Object.getOwnPropertyNames(this)`, jsGlobals);
+
+const windowGlobals = new Promise((resolve, reject) => {
+  https.get('https://tidoust.github.io/reffy-reports/whatwg/crawl.json',
```

Comment:
Sigh, Node.js's built-in HTTP library is so terrible. If we do this more than once, let's use a library. But I think this code is good for now; the cost of adding a library dependency is not worth cleaning it up.

Reply:
SGTM, it is a bit of a mess